src/test/scala/org/clulab/habitus/variables/TestEntities.scala

```diff
@@ -91,82 +98,422 @@ class TestEntities extends Test {
+      Seq(
+        "Yield" -> Seq("yields"),
+        "DrySeason" -> Seq("dry season"),
+        "Date" -> Seq("2021", "2019"),
+        "Quantity" -> Seq("between 6.5 and 7.5 t ha-1", "between 5.0 and 6.0 t ha-1")
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent33",
+      "seeds of the rice variety Sahel 108 are sown, a short- cycle variety (around 125 days)",
+      Seq(
+        "Quantity" -> Seq("125 days"),
+        "Planting" -> Seq("sown"),
+        "GenericCrop" -> Seq("variety", "variety", "seeds"),
+        "Crop" -> Seq("rice", "Sahel 108")
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent34",
+      "Broadcast seeding is carried out by hand on irrigated plots with a 2-5 cm depth sheet of water",
+      Seq(
+        "Quantity" -> Seq("2-5 cm"),
+        "Planting" -> Seq("seeding"),
+        "GenericCrop" -> Seq("seeding")
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent35",
+      "only two top dressings are applied in the CONV scenario ( the first one with urea and dia- mmonium phosphate (DAP) at the beginning of tillering; and the second only with urea at panicle initiation.",
+      Seq(
+        "Fertilizer" -> Seq("urea", "DAP", "phosphate", "urea"),
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent36",
+      "In INT, a basic dressing is firstly applied with urea and DAP, followed by two top-dressing applications with urea.",
+      Seq(
+        "Fertilizer" -> Seq("urea", "DAP", "urea"),
+      )
+    ),
 
+    VariableTest(
+      passingTest,
+      "sent37",
+      "average yields for the two seasons assessed of 4832 kg ha-1 and 7425 kg ha-1 for the areas under CONV and INT management, respectively.",
+      Seq(
+        "Yield" -> Seq("yields"),
+        "Quantity" -> Seq("4832 kg ha-1", "7425 kg ha-1")
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent38",
+      "Agricultural inputs, machinery operations and yields considered in the four scenarios evaluated: conventional (CONV), intensive (INT), and the two refernce scenarios SAED_2td and SAED_3td. DAP: Diammonium phosphate.",
+      Seq(
+        "FertilizerUse" -> Seq("inputs"),
+        "Yield" -> Seq("yields"),
+        "Fertilizer" -> Seq("DAP", "Diammonium phosphate")
+      )
+    ),
+    VariableTest(
+      passingTest,
+      "sent39",
+      "Fertilisers considered and associated nutrients in the four scenarios evaluated: conventional (CONV), intensive (INT), and the two reference scenarios SAED_2td and SAED_3td. DAP: diammonium phosphate.",
+      Seq(
+        "Fertilizer" -> Seq("DAP", "diammonium phosphate")
+      )
+    ),
+    VariableTest(
+      failingTest,
+      "sent40",
+      "In plots receiving fertilizer, DAP was applied basally (19.3 and 21.5 kg N and P ha−1 ), and three urea splits were broadcasted into 1–5 cm of water (101.3 kg N ha−1 ; 40% at early-tillering, 40% at panicle initiation, and 20% at heading)",
```

Comment:
to be honest, I am not sure what the ideal quantity extraction should be: 19.3 kg N and P ha-1 and 21.5 kg N and P ha-1? 19.3 kg ha-1 and 21.5 kg ha-1? This test will probably remain forever failing :D 
I think it is fine to leave it as you have it now :)

Reply:
It's interesting to see what you have to deal with.  It looks like they are applying nitrogen at the rate of 19.3 kg/ha and phosphorus at 21.5 kg/ha.

Reply:
That was my first interpretation but I was led astray by that 'and' between N and P (instead of something like 19.3 kg N and 21.5 kg P). With your comment, I am now sure that is the right interpretation.